<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anisotropy window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background: rgb(214, 219, 233,255);
}

QListView{
background: rgb(214, 219, 233,255);
border: 2px solid rgb(231, 236, 250, 255);
}

QPushButton:hover{
background: rgb(255, 255, 191,255);
border:  2px solid rgb(255, 170, 0,255);
}

QPushButton{
border:  2px solid rgb(231, 236, 250, 255);;
background: rgb(214, 219, 233,255);
}

QTableWidget{
background: rgb(214, 219, 233,255);
border: 2px solid rgb(231, 236, 250, 255);
}

QHeaderView::section{
background-color: rgb(235, 241, 255, 255);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,7,1">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="5,9">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="textEditHexagonal">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomTableWidget" name="tableWidgetHexagonal">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,7,3,2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="textEditUniaxial">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listViewUniaxial">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomTableWidget" name="tableWidgetUniaxial">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTableWidget</class>
   <extends>QTableWidget</extends>
   <header>CustomTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>on_add_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>on_delete_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>on_ok_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>692</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>635</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_add_clicked()</slot>
  <slot>on_delete_clicked()</slot>
  <slot>on_ok_clicked()</slot>
 </slots>
</ui>
